virttest/utils_test/__init__.py

```diff
@@ -775,14 +782,22 @@ def transfer_data(session, host_cmd, guest_cmd, n_time, timeout,
     host_script = params.get("host_script", "serial_host_send_receive.py")
     host_script = os.path.join(data_dir.get_root_dir(), "shared", "deps",
                                "serial", host_script)
-    host_cmd = "python %s -s %s -f %s -a %s" % (host_script, host_device,
-                                                host_data_file, action)
+
+    host_python = find_host_python()
```

Comment:
An alternative approach is using an inline shell statement `$(command -v python3 python | head -1)`, but this assumes we are running in bash.